src/parser.rs

```diff
@@ -3588,5 +3628,47 @@ mod test {
         } else {
           panic!("Failed to read fake env float");
         }
+
```

Comment:
Looks good now :)

About the error, it's something we have to think about, but I guess you're right. Returning an empty value is what bash does anyway, so for the sake of consistency maybe we should do that as well.

In either case (and on a totally unrelated note), the library definitely needs better error handling / reporting (especially now that nom improved on error support too).

Thanks for your contributions!